### ISSUE TITLE
Importing many cards to a new GitHub repo triggers abuse detection

### ISSUE THREAD

**Issue by nickserv** [RickyCook/import-trello-github#5]
```
ERROR 2016-12-12 22:13:37,881 [github] :: You have triggered an abuse detection mechanism and have been temporarily blocked from content creation. Please retry your request again later.
```

Unfortunately I don't seem to be able to give a different kind of token to prevent this, and because issues can't be deleted I'll have to do the rest of my import manually.

**Comment by RickyCook** [comment]
I haven't really used this in a while, so correct me if I'm wrong but I think that the `--statedir` argument should allow you to continue an import? But you do need to use it right from the start :(

**Comment by nickserv** [comment]
Oh good point, I didn't try that. I ended up doing the rest manually, fortunately it only skipped my archived cards. Thanks for the advice though.

Is this using an oauth token or key that gives it the more relaxed rate limiting? If not, it would help.

**Comment by RickyCook** [comment]
it doesn't, but it tracks the cards that've already been imported so should start from where the last card failed

**Comment by nickserv** [comment]
My problems are resolved anyway since I finished my only import. Want to leave this open or close it and have people use `--statedir` as a workaround?

**Comment by RickyCook** [comment]
Yeah; it's certainly not resolved. Potentially I should force `--statedir` as non-optional so that nobody accidentally hits this with no way out
